client/src/api/member.js

```diff
@@ -1,3 +1,5 @@
+//get functions
+
 export async function getAllMember(ID) {
```

Comment:
```suggestion
export async function getAllMember(id) {
```

variables like this should be in camelCase